### PR TITLE
Add a troubleshooting section.

### PR DESCRIPTION
**Pull request by gauravgahlot**
## Description

The PR adds a troubleshooting section to the README.

It also adds a solution for the following commonly faced error:

```
Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): resource "scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68" not found in file cache
```

I also logged an issue for the same problem: #100 

Signed-off-by: Gaurav Gahlot <gauravgahlot0107@gmail.com>

### CODE REVIEW

**Comment by gauravgahlot** [comment]
@googlebot I signed it!

**Comment by LisaFC** [comment]
This is great but I think the solution is a little too specific in terms of the approach - both in terms  of how to install and how to run Hugo (I think being able to install the extended version hugo via npm is *extremely* new, and I think most users run `hugo` directly). I'll add some suggestions.

I will definitely add installing Hugo via npm to the user guide, thanks for letting me know about that!

File: README.md [hunk 1]
```diff
@@ -87,3 +87,59 @@ To delete the container run:
...
+```
+
+To solve this error, first install `hugo-extended` as:
+
+```
+npm install hugo-extended --save-dev
+# or...
+yarn add hugo-extended --dev
+```
+
```
Comment (LisaFC) [inline README.md:118]: I think from this line down we've moved out of the realm of troubleshooting (or even anything Docsy-specific) and just telling users how to run hugo using npm and their `package.json` file, which may not be the preferred approach of all our users.

File: README.md [hunk 2]
```diff
@@ -87,3 +87,59 @@ To delete the container run:
...
+```
+➜ hugo server
+
+INFO 2021/01/21 21:07:55 Using config file: 
+Building sites … INFO 2021/01/21 21:07:55 syncing static files to /
+Built in 288 ms
+Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): resource "scss/scss/main.scss_9fadf33d895a46083cdd64396b57ef68" not found in file cache
+```
+
+To solve this error, first install `hugo-extended` as:
```
Comment (LisaFC) [inline README.md:111]: This error occurs if you have not installed the **extended** version of Hugo.  See our [user guide](https://www.docsy.dev/docs/getting-started/#prerequisites-and-installation) for instructions on how to install Hugo.

File: README.md (hunk 1 above)
Comment (LisaFC) [reply README.md:118]: This is still very useful content though so I'm going to add it to our user guide rather than having it here.

**Comment by LisaFC** [comment]
See https://github.com/google/docsy/pull/434

**Comment by gauravgahlot** [comment]
> See [google/docsy#434](https://github.com/google/docsy/pull/434)

I see the PR merged. What happens to this one now?

**Comment by LisaFC** [comment]
> > See [google/docsy#434](https://github.com/google/docsy/pull/434)
> 
> I see the PR merged. What happens to this one now?

So the PR is in to the actual user guide for Docsy - I still think it's useful to have the troubleshooting info in the example site README as well, we can just cut it down a bit. So if you change line 111 as I suggested, then delete everything from there onwards, we can merge this in to the README.

**Comment by gauravgahlot** [comment]
@LisaFC I think I messed up. I forgot to pull your commit and force pushed the branch.
Can you please apply your changes again?

**Comment by LisaFC** [comment]
No problem!

**Pull request merged**